napari/utils/events/evented_model.py

```diff
@@ -126,33 +157,48 @@ def __init__(self, **kwargs):
 
         self._events.source = self
         # add event emitters for each field which is mutable
-        fields = []
-        for name, field in self.__fields__.items():
-            if field.field_info.allow_mutation:
-                fields.append(name)
-        self._events.add(**dict.fromkeys(fields))
+        event_names = [
+            name
+            for name, field in self.__fields__.items()
+            if field.field_info.allow_mutation
+        ]
+        event_names.extend(self.__property_setters__)
+        self._events.add(**dict.fromkeys(event_names))
+
+    def _super_setattr_(self, name: str, value: Any) -> None:
```

Comment:
Is it standard to add the type annotation `-> None` to methods that don't return anything? Intuitively, I think that it's better to omit it because I see it as easily implied. But if being explicit here is desired/standard, I don't mind it - just want to know best practices for other type annotated parts of the code.

Reply:
lacking a type hint, the implication is ` -> Any:`

Reply:
(strict mode in mypy requires it, and I generally find it useful in my linting)